aspnetcore/signalr/streaming.md

```diff
@@ -133,6 +133,25 @@ To end the stream from the client, call the `dispose` method on the `ISubscripti
 
 ::: moniker-end
 
+::: moniker range=">= aspnetcore-3.0"
+## Java client
+The SignalR Java client uses the `stream` method to invoke streaming methods. It accepts three arguments:
```

Comment:
Should this say "three or more arguments", i.e., could the hub method take more than 1?